int8 gear
int8 speed
int16 steer
int16 brake
